apps/freelance/src/app/app.tsx

```diff
@@ -1,25 +1,21 @@
-import styled from 'styled-components';
+import { StyledApp } from './app.styled';
 import { BrowserRouter, Route, Routes } from 'react-router-dom';
-import ExamplePage from '../pages/example-page/example-page';
 import store from '../redux/example-store';
 import { Provider } from 'react-redux';
+import LoginPage from '../pages/login-page/login-page';
```

Comment:
absolute path

Reply:
Fixed. Page is imported with absolute path.
```import LoginPage from '@pages/login-page/login-page';```